.github/workflows/main.yml

```diff
@@ -266,33 +222,24 @@ jobs:
         shell: bash -l {0}
         run: |
           source $HOME/.cargo/env
-          make SUBSTRATE_PARSER_FULL=1 DISABLE_PREVIOUS=1
-          mv ./app/pkg/installer_s.sh ./app/pkg/installer_nanos_xl_current.sh
+          make SUBSTRATE_PARSER_FULL=1
```

Comment:
shouldn't we keep DISABLE_PREVIOUS flag here?

Reply:
we're not using that currently but we can keep that for the future 👌🏻